frontend/app/mocks/db.ts

```diff
@@ -113,11 +121,40 @@ db.subscription.create({
 
 db.subscription.create({
   id: '0000002',
-  sin: '800000002',
+  sin: '800011819',
   email: 'tester@example.com',
   subscribed: true,
   preferredLanguage: '1033', // "English", @see ~/resources/power-platform/preferred-language.json
   alertType: 'ei',
 });
+db.subscriptionConfirmationCode.create({
+  id: '0000001',
+  email: 'user@example.com',
+  confirmationCode: '0001',
+  createdDate: new Date(new Date().getTime() - 12 * 24 * 60 * 60 * 1000),
+  expiryDate: new Date(new Date().getTime() - 10 * 24 * 60 * 60 * 1000), //Current date + 2 days
+});
+db.subscriptionConfirmationCode.create({
+  id: '0000002',
+  email: 'user@example.com',
+  confirmationCode: '1234',
+  createdDate: new Date(new Date().getTime()),
+  expiryDate: new Date(new Date().getTime() + 2 * 24 * 60 * 60 * 1000), //Current date + 2 days
+});
+db.subscriptionConfirmationCode.create({
+  id: '0000003',
+  email: 'user@example.com',
+  confirmationCode: '1001',
+  createdDate: new Date(new Date().getTime() - 8 * 24 * 60 * 60 * 1000),
+  expiryDate: new Date(new Date().getTime() - 6 * 24 * 60 * 60 * 1000), //Current date + 2 days
```

Comment:
Did you intend on having these comments read "created date + 2 days" rather than "Current date + 2 days"?

Reply:
yes, the text has been changed

Reply:
Text has been changed, but some comments don't represent the actual value. 

For example:
![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/55502055/b14f20e0-e268-4a7c-96fc-60e23372b25a)

This should read as current date and not created date:
```typescript
  createdDate: new Date(new Date().getTime() - 12 * 24 * 60 * 60 * 1000), // current date - 12 days
  expiryDate: new Date(new Date().getTime() - 10 * 24 * 60 * 60 * 1000), // current date - 10 days
```

Reply:
should be current date in the comments now instead of created date